<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="0">
     <widget class="QRadioButton" name="rbtn_musical">
      <property name="styleSheet">
       <string notr="true">font: 700 20pt &quot;Segoe UI&quot;;
color: rgb(255, 0, 127);</string>
      </property>
      <property name="text">
       <string>Musical_instruments</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="8" column="2" colspan="2">
     <widget class="QRadioButton" name="rbtn_easy">
      <property name="styleSheet">
       <string notr="true">font: 700 20pt &quot;Segoe UI&quot;;
color: rgb(255, 0, 0);
background-color: rgb(0, 0, 0);
</string>
      </property>
      <property name="text">
       <string>easy</string>
      </property>
     </widget>
    </item>
    <item row="12" column="2" colspan="2">
     <widget class="QRadioButton" name="rbtn_hard">
      <property name="styleSheet">
       <string notr="true">font: 700 20pt &quot;Segoe UI&quot;;
color: rgb(255, 0, 0);
background-color: rgb(0, 0, 0);
</string>
      </property>
      <property name="text">
       <string>hard</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QRadioButton" name="rbtn_sports">
      <property name="styleSheet">
       <string notr="true">font: 700 20pt &quot;Segoe UI&quot;;
color: rgb(255, 0, 127);</string>
      </property>
      <property name="text">
       <string>Sports</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="11" column="2" colspan="2">
     <widget class="QRadioButton" name="rbtn_medium">
      <property name="styleSheet">
       <string notr="true">font: 700 20pt &quot;Segoe UI&quot;;
color: rgb(255, 0, 0);
background-color: rgb(0, 0, 0);
</string>
      </property>
      <property name="text">
       <string>medium</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="QTextBrowser" name="text">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0, 127);</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9.75pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:24pt; font-weight:700;&quot;&gt;welcam to gam Han Gman&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0">
     <widget class="QRadioButton" name="rbtn_clothes">
      <property name="styleSheet">
       <string notr="true">font: 700 20pt &quot;Segoe UI&quot;;
color: rgb(255, 0, 127);</string>
      </property>
      <property name="text">
       <string>Clothes</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="7" column="0" rowspan="2">
     <widget class="QRadioButton" name="rbtn_animals">
      <property name="styleSheet">
       <string notr="true">font: 700 20pt &quot;Segoe UI&quot;;
color: rgb(255, 0, 127);</string>
      </property>
      <property name="text">
       <string>Animals</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="9" column="0" rowspan="3">
     <widget class="QRadioButton" name="rbtn_colors">
      <property name="styleSheet">
       <string notr="true">font: 700 20pt &quot;Segoe UI&quot;;
color: rgb(255, 0, 127);</string>
      </property>
      <property name="text">
       <string>Colors</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="13" column="1" colspan="3">
     <widget class="QPushButton" name="btn_check">
      <property name="styleSheet">
       <string notr="true">font: 700 20pt &quot;Segoe UI&quot;;
background-color: qconicalgradient(cx:0, cy:0, angle:14.8, stop:0 rgba(0, 0, 0, 0));
color: rgb(255, 0, 127);
</string>
      </property>
      <property name="text">
       <string>check</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>561</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
